### PR TITLE
fix: Wrong link to documentation in readme

### PR DESCRIPTION
**Pull request by bhattabhi013**
## Description

Fix issue #106: Wrong link to documentation in the readme.
Changed the link from `https://firebase.flutter.dev/docs/cli/` to `https://firebase.google.com/docs/flutter/setup`.

## Type of Change

- [ ] ✨ `feat` -- New feature (non-breaking change which adds functionality)
- [x] 🛠️ `fix` -- Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ `!` -- Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 `refactor` -- Code refactor
- [ ] ✅ `ci` -- Build configuration change
- [ ] 📝 `docs` -- Documentation
- [ ] 🗑️ `chore` -- Chore

### CODE REVIEW

**Comment by bhattabhi013** [comment]
@Lyokone , Can you please review this one?

**Review by Lyokone — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 </p>
 
 <p align="center">
-  <a href="https://firebase.flutter.dev/docs/cli">Documentation</a> &bull; 
+  <a href="https://firebase.google.com/docs/flutter/setup?platform=ios">Documentation</a> &bull;
```
[the remark this answers is not in the document]
Comment (bhattabhi013) [reply README.md]: Thanks for reviewing and suggestions.
I've done the needful.

**Review by Lyokone — APPROVED** [review]

**Review by Ehesp — APPROVED** [review]

**Comment by bhattabhi013** [comment]
Hy maintainers,
Does it come under hacktoberfest?

**Comment by Lyokone** [comment]
Hello @bhattabhi013,
This PR does not qualify for hacktoberfest, see here for more information: https://invertase.io/community/hacktoberfest
Thanks for your contribution :)

**Pull request merged**